src/lorawan.cpp

```diff
@@ -219,7 +219,11 @@ void lora_send(void *pvParameters) {
       xQueueReceive(LoraSendQueue, &SendBuffer, (TickType_t)0);
       break;
     case LMIC_ERROR_TX_BUSY:   // LMIC already has a tx message pending
+      ESP_LOGI(TAG, "Message not sent, LMIC busy, will retry later");
```

Comment:
There is intentionally no info log at this point in the code, because it would repeat log messages every second while the lmic state engine remains in this state.

Reply:
PS: perhaps use log level verbose here